app/_locales/es/messages.json

```diff
@@ -1,10 +1,840 @@
+  },
   "appName": {
     "message": "MetaMask",
-    "description": "The name of the application"
+    "description": "El nombre de la aplicacion"
   },
-  "appDescription": {
-    "message": "Administración de identidad en Ethereum",
-    "description": "The description of the application"
+  "attemptingConnect": {
+    "message": "Intentando conectar a la Blockchain"
+  },
+  "available": {
+    "message": "Disponible"
+  },
+  "back": {
+    "message": "Atras"
+  },
+  "balance": {
+    "message": "Balance:"
+  },
+  "balances": {
+    "message": "Tus balances"
+  },
+  "balanceIsInsufficientGas": {
+    "message": "Insuficiente balance para el gas total actual"
+  },
+  "beta": {
+    "message": "BETA"
+  },
+  "betweenMinAndMax": {
+    "message": "Debe ser mayor o igual a $1 y menor o igual a $2",
+    "description": "helper para ingresar hex como un ingreso decimal"
+  },
+  "borrowDharma": {
+    "message": "Pedir prestado con Dharma (Beta)"
+  },
+  "buy": {
+    "message": "Comprar"
+  },
+  "buyCoinbase": {
+    "message": "Comprar en Coinbase"
+  },
+  "buyCoinbaseExplainer": {
+    "message": "Coinbase es la forma mas popular en el mundo para comprar y vender bitcoin, ethereum, y litecoin."
+  },
+  "cancel": {
+    "message": "Cancelar"
+  },
+  "clickCopy": {
+    "message": "Click para copiar"
+  },
+  "confirm": {
+    "message": "Confirmar"
+  },
+  "continue": {
+    "message": "Continuar"
+  },
+  "confirmContract": {
+    "message": "Confirmar contracto"
+  },
+  "confirmPassword": {
+    "message": "Confirmar contraseña"
+  },
+  "enterPasswordConfirm": {
+    "message": "Ingresa tu password para confirmar"
+  },
+  "confirmTransaction": {
+    "message": "Confirmar transaccion "
+  },
+  "continueToCoinbase": {
+    "message": "Continuar a Coinbase"
+  },
+  "contractDeployment": {
+    "message": "Desplegar contrato"
+  },
+  "currentConversion": {
+    "message": "Conversión Actual"
+  },
+  "conversionProgress": {
+    "message": "Conversión en progreso"
+  },
+  "copiedButton": {
+    "message": "Copiado"
+  },
+  "copiedClipboard": {
+    "message": "Copiado al portapapeles"
+  },
+  "copiedExclamation": {
+    "message": "Copiado!"
+  },
+  "copy": {
+    "message": "Copiar"
+  },
+  "copyToClipboard": {
+    "message": "Copiar al portapapeles"
+  },
+  "copyButton": {
+    "message": " Copiar "
+  },
+  "copyPrivateKey": {
+    "message": "Esta es tu llave privada(Click para copiar)"
+  },
+  "create": {
+    "message": "Crear"
+  },
+  "createAccount": {
+    "message": "Crear Cuenta"
+  },
+  "createDen": {
+    "message": "Crear"
+  },
+  "crypto": {
+    "message": "Crypto",
+    "description": "Exchange type (cryptocurrencies)"
+  },
+  "customGas": {
+    "message": "Customizar Gas"
+  },
+  "customize": {
+    "message": "Customizar"
+  },
+  "currentRPC": {
+    "message": "RPC actual"
+  },
+  "customRPC": {
+    "message": "RPC Personalizado"
+  },
+  "newRPC": {
```

Comment:
Not here https://github.com/MetaMask/metamask-extension/blob/master/app/_locales/en/messages.json @danfinlay